x-pack/legacy/plugins/license_management/public/register_route.ts

```diff
@@ -87,6 +87,7 @@ if (licenseManagementUiEnabled) {
                   const unmountApp = await app.mount({ ...npStart } as any, {
                     element,
                     appBasePath: '',
+                    onAppLeave: () => undefined,
                   });
                   manageAngularLifecycle($scope, $route, unmountApp as any);
```

Comment:
TIL this seems to be the standard way to mount app in shims. Add to had a noop `onAppLeave` on all of them. The feature is specified to be working only with NP plugin's applications so I don't think this is an issue.